integration-test/.env.example

```diff
@@ -0,0 +1 @@
+API_HOST="https://wcc-backend.fly.dev"
```

Comment:
Could you use localhost:8080 as default template? I didn't want to expose our dev env.

Reply:
Good point! Made a fix